code/modules/holiday/holidays.dm

```diff
@@ -586,6 +586,14 @@
 /datum/holiday/hotdogday/greet()
 	return "Happy National Hot Dog Day!"
 
+/datum/holiday/indigenousUN //Indigenous Peoples' Day from Earth!
```

Comment:
dont use uppercase, and it can stay indigenous anyway

Reply:
done